content/en/blog/SSI-identity-provider/index.md

```diff
@@ -0,0 +1,62 @@
+---
+date: 2022-04-07
+title: "SSI-Empowered Identity Provider"
+linkTitle: "SSI-Empowered Identity Provider"
+description: "Open ID Connect (OIDC) is a popular identity protocol for authenticating users and providing identity data for access control. It allows web services to externalize the authentication of end-users by securely signing users in using a third-party identity provider. Findy Agency team has experimented with integrating SSI (self-sovereign identity) agent capabilities to a sample OIDC provider, thus enabling verified data usage in the login flow. The proof-of-concept shows that this approach would allow numerous web applications to switch to SSI-based login with minimal changes."
+author: Laura Vuorenoja
+resources:
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+
+Utilizing SSI wallets and verifiable credentials in OIDC authentication flows has been an interesting research topic for [our team](https://findy-network.github.io) already for a while now. As said, [the OIDC protocol](https://openid.net/connect/) is popular. Countless web services sign their users in using OIDC identity providers. And indeed, it provides many benefits, as it simplifies the authentication for service developers and end-users. The developers do not have to reinvent the authentication wheel and worry about storing username/password information. The users do not have to maintain countless digital identities with several different passwords.
+
+{{< figure src="https://pbs.twimg.com/media/FA_yO0jUcAM2fMa.jpg" width="500px" attr="<small>Example of login page with support for multiple identity providers.<br/>Image source: IndieWeb NASCAR Problem</small>" attrlink="https://indieweb.org/NASCAR_problem" >}}
+
+However, [the protocol is not flawless](https://medium.com/mattr-global/if-you-build-an-island-youll-need-a-boat-537f48525edc), and it seems evident that using verified data would fix many of the known weaknesses.
+
+**Our most significant concerns for current OIDC protocol practices are related to privacy.**
+
+Let's suppose that our imaginary friend Alice uses an application, say a dating service, that provides a Facebook login button for its users. Each time Alice logs in, Facebook becomes aware that Alice uses the dating service. It also learns at which time and how often Alice is using the service. Alice probably didn't want to share this data with Facebook and did this unintentionally. Even worse, Alice probably uses a similar login approach with other applications. Little by little, Facebook learns a great deal about which applications Alice is using and how often. Moreover, as applications usually provide a limited amount of login options, most users choose the biggest identity providers such as Facebook and Google. The big players end up collecting an enormous amount of data over users.
```

Comment:
The whole chapter is excellent! One question though:
- if long time JWT is used, is it really possible to track how Alice is using the service, e.g. how often, etc.